plugins/modules/keycloak_authentication.py

```diff
@@ -14,10 +14,11 @@
 short_description: Configure authentication in Keycloak
 
 description:
-    - This module actually can only make a copy of an existing authentication flow, add an execution to it and configure it.
-    - It can also delete the flow.
+    - This module supports creation, making a copy, deletion and configuration of an authentication flow.
+    - It can also add or update an authentication execution to a flow (step or sub-flow). Deletion is not supported.
+    - This module also supports registration, update and deletion of an authentication required action.
 
-version_added: "3.3.0"
+version_added: "7.0.0"
```

Comment:
Please revert this change. The module was added in version 3.3.0, not in 7.0.0.

Reply:
My bad, done.